.eslintrc.js

```diff
@@ -276,7 +276,7 @@ module.exports = {
 		'no-template-curly-in-string': 'warn',
 		'no-this-before-super': 'warn',
 		'no-throw-literal': 'warn',
-		'no-undef': 'error',
+		'no-undef': 'off',
```

Comment:
Nit: Arguably this would be better as a `warn`, we do generally want to clean those up so disabling it entirely feels wrong